scaladoc/test/dotty/tools/scaladoc/signatures/SignatureTest.scala

```diff
@@ -110,6 +116,11 @@ abstract class SignatureTest(
 
     def processFile(path: Path): Unit = if filterFunc(path) then
       val document = Jsoup.parse(IO.read(path))
+      val documentable = document.select(".groupHeader").forEach { element =>
+        val signature = element.select(".groupHeader").eachText.asScala.mkString("")
+        val all = s"$signature"
+        signatures += all
```

Comment:
I'm not 100% on what this is actually doing or why it's needed. Could you explain it?

Reply:
So from what I understand how the test is done.
If we take the example of how to test a method `foo1`.
The function `signaturesFromDocumentation` will create a documentation and then from this documentation the function `processFile` will retrieve the different parts of this function foo1 thanks to the different html class, for example the name value by retrieving `foo1` from the class `.header .documentableName`. These different values are then added to the `val signature: Builder[String, List[String]]`. So I needed a way to retrieve the different parts of an extension's signature in this documentation. So I get in each extension's `groupHeader` class all that `groupHeader`, to get the signature. Is this what you wanted to know?